src/NetworkPort.php

```diff
@@ -1276,7 +1279,21 @@ protected function showPort(array $port, $dprefs, $so, $canedit, $agg, $rand, $w
                             }
                             break;
                         default:
-                            $output .= $port[$option['field']];
+                            if ($option['table'] == $this->getTable()) {
+                                $output .= $port[$option['field']];
+                            } else {
+                                $iterator = $DB->request([
+                                    'SELECT' => $option['field'],
+                                    'FROM'   => $option['table'],
+                                    'WHERE'  => [
+                                        'itemtype' => $this->getType(),
+                                        'items_id' => $port['id']
```

Comment:
I guess that plugins may add search options that points to a table that does not contains `itemtype`/`items_id` fields.